src/views/home/sections/Sponsors/Sponsors.tsx

```diff
@@ -0,0 +1,46 @@
+import Image from "next/image";
+import Link from "next/link";
+
+import { Col, Row } from "react-bootstrap";
+
+import sponsors from "assets/data/sponsors.json";
+
+import styles from "./Sponsors.module.scss";
+
+import stickerMule from "assets/sponsors/sticker_mule.svg";
+
+function SponsorItem(props: any) {
+	return (
+		<Link href={props.data.url}>
```

Comment:
Suggestion: `next/link` should be used only for internal routing, not to link to external sites.